openapi/components/schemas/OneTimeSale.yaml

```
@@ -0,0 +1,93 @@
+type: object
+required:
+  - type
+  - planId
+properties:
+  id:
+    type: string
+    description: ID of the order item.
+    readOnly: true
+    maxLength: 50
+    example: ord_01GYJPRKHBD6ZYHH897QCJMBS4
+  type:
+    description: Specifies the type of order.
+    type: string
+    x-basic: true
+    enum:
+      - one-time-sale
+  status:
+    description: Status of the one-time order.
```

question: should this be 'sale'?
```suggestion
    description: Status of the one-time sale.
```

---

Yes, thanks for spotting.